### ISSUE TITLE
[20210516] Speed preview is broken after slice

### ISSUE THREAD

**Issue by d-schmidt** [smartavionics/Cura#111]
**Application version**
20210516

**Platform**
win 10 64

**Printer**
ender 3v2

**Reproduction steps**
slice with speed active
dis/enable any line type in speed drop down to get back correct view

**Screenshot(s)**
![grafik](https://user-images.githubusercontent.com/3226483/118413638-a5f12900-b6a0-11eb-9a3a-465a9e6f26a6.png)

**Actual results**
10-0mm/s all blue

**Expected results**
colors

**Project file**
-

**Log file**
-

**Additional information**
-

**Comment by 5axes** [comment]
Same issue for me : 
**Application version** 20210516

**Platform** win 10 64

The maximum speed value is not correctly initialized after a calculation, an option must be changed to force the change.

![image](https://user-images.githubusercontent.com/11015345/118414339-3b41ec80-b6a4-11eb-8c53-9a31937d8a87.png)

**Comment by smartavionics** [comment]
Yes, UM have made some changes in this area, I think this is an upstream bug.
